sharding/client/client.go

```diff
@@ -239,3 +240,7 @@ func initSMC(c *shardingClient) (*contracts.SMC, error) {
 	}
 	return contract, nil
 }
+
```

Comment:
Can we add an overall function comment for `DepositFlagSet()`? 
Everything else looks good

Reply:
Yeah good thinking